roles/container-engine/docker/tasks/main.yml

```diff
@@ -101,6 +101,12 @@
     dest: "{{ yum_repo_dir }}/docker.repo"
   when: ansible_distribution == "Fedora" and not is_atomic
 
+- name: Remove legacy docker repo file
```

Comment:
Can you move this task to roles/container-engine/docker/tasks/pre-upgrade.yml? That file exists in most roles to handle exactly this type of task.

Reply:
moved